lang/pt-br.xml

```diff
@@ -215,10 +215,10 @@ depending on its size and your hardware.</app_delete_note>
 	</compatibility>
 
 	<content_manager name="Gerenciador de conteúdo">
-		<search>Search</search>
+		<search>Pesquisar</search>
 		<application name="Aplicativos">
 			<delete>Os aplicativos selecionados e todos os dados relacionados, inclusive dados salvos, serão excluídos.</delete>
-			<no_item>Não há nenhum itens de conteúdo.</no_item>
+			<no_item>Não há nenhum item de conteúdo.</no_item>
```

Comment:
revert it, it is based on psvita system language.

Reply:
what do i need to revert exactly?

Reply:
this line

Reply:
seems like he already did it, can we get this merged or just close this altogether? @Croden1999 @Zangetsu38

Reply:
> seems like he already did it, can we get this merged or just close this altogether? @Croden1999 @Zangetsu38

rebase and push

Reply:
done, i think so.

Reply:
I think i will just close this and open a new one with more cleaner tree anyways.